build-gluster-org/scripts/centos-regression.sh

```diff
@@ -117,20 +117,14 @@ else
 fi
 
 RET=$?
-if [ $RET = 0 ]; then
-    V="+1"
-    VERDICT="SUCCESS"
-else
-    V="-1"
-    VERDICT="FAILED"
-fi
-
 if [ ${RET} -ne 0 ]; then
+    # Update Gerrit with the success/failure status
+    sudo mv /tmp/gluster_regression.txt $WORKSPACE || true
```

Comment:
Why || true ? It if fail, we want to see it, no ?

Reply:
Ok, didn't see it was here before.

Reply:
It would fail when the test suite passes as that time the file will not be created.